source/extensions/common/wasm/ext/envoy_proxy_wasm_api.cc

```diff
@@ -33,3 +33,7 @@ extern "C" PROXY_WASM_KEEPALIVE void envoy_on_resolve_dns(uint32_t context_id, u
                                                           uint32_t data_size) {
   getEnvoyContextBase(context_id)->onResolveDns(token, data_size);
 }
+
+extern "C" PROXY_WASM_KEEPALIVE void envoy_on_stat(uint32_t context_id, uint32_t data_size) {
```

Comment:
I'm not the fan of this function name, since it doesn't really say what's happening.

Perhaps `envoy_on_stats_update` would make more sense? What do you think?

Reply:
envoy_on_stats_update makes sense to me